sympy/tensor/array/expressions/__init__.py

```diff
@@ -0,0 +1,108 @@
+r"""
+Array expressions are expressions representing N-dimensional arrays, without evaluating them.
+They are *lazy* as they don't evaluate the operations on the arrays until required and
+can define operations on unknown array (in a certain way reminiscent of graphs of
+operations in TensorFlow).
```

Comment:
While internally TensorFlow still has these graphs, since the TF2 release, the API exposes it as "eager" tensors, meaning they appear to not be evaluated lazily (eventhough they really are). So this comparison might be confusing to a TF user who is not familiar with TF1 or not very knowledgable about how TF2 works internally (really, TF1 and TF2 should be considered two different libraries in my opinion).